website/docs/aiml/inferentia/inference.md

```diff
@@ -0,0 +1,114 @@
+
+The lab uses Karpenter to provision an Inferentia node. Karpenter can detect the pending pod which requires Neuron cores and launch an inf1 instance which has the required Neuron cores.
+
+:::tip
+You can learn more about Karpenter in the [Karpenter module](../../autoscaling/compute/karpenter/index.md) that's provided in this workshop.
+:::
+
+Karpenter has been installed in our EKS cluster, and runs as a deployment:
+
+```bash
+$ kubectl get deployment -n karpenter
+NAME        READY   UP-TO-DATE   AVAILABLE   AGE
+karpenter   2/2     2            2           105s
+```
+
+The only setup that we will need to do is to update our EKS IAM mappings to allow Karpenter nodes to join the cluster:
+
+```bash
+$ eksctl create iamidentitymapping --cluster $EKS_CLUSTER_NAME \
+    --region=$AWS_REGION --arn $KARPENTER_NODE_ROLE \
+    --group system:bootstrappers --group system:nodes \
+    --username system:node:{{EC2PrivateDNSName}}
+```
+
+Karpenter requires a provisioner to provision nodes. This is the Karpenter provisioner that we will create:
+
+```file
+manifests/modules/aiml/inferentia/provisioner/provisioner.yaml
+```
+
+Apply the provisioner manifest:
+
+```bash
+$ kubectl apply -k ~/environment/eks-workshop/modules/aiml/inferentia/provisioner/
+```
+
+### Create a pod for inference
+
+Now we can deploy a Pod for inference:
+
+```bash
+$ kubectl apply -k ~/environment/eks-workshop/modules/aiml/inferentia/inference/
+```
+
+Karpenter detects the pending pod which needs Neuron cores and launches an inf1 instance which has the Inferentia chip. Monitor the instance provisioning with the following command:
+
+```bash test=false
+$ kubectl logs -f -n karpenter deploy/karpenter -c controller
+
+2022-10-28T08:24:42.704Z        DEBUG   controller.provisioning.cloudprovider   Created launch template, Karpenter-eks-workshop-cluster-3507260904097783831  {"commit": "37c8653", "provisioner": "default"}
+2022-10-28T08:24:45.125Z        INFO    controller.provisioning.cloudprovider   Launched instance: i-09ddba6280017ae4d, hostname: ip-100-64-10-250.ap-northeast-1.compute.internal, type: inf1.xlarge, zone: ap-northeast-1a, capacityType: spot  {"commit": "37c8653", "provisioner": "default"}
+2022-10-28T08:24:45.136Z        INFO    controller.provisioning Created node with 1 pods requesting {"aws.amazon.com/neuron":"1","cpu":"125m","pods":"6"} from types inf1.xlarge, inf1.2xlarge, inf1.6xlarge, inf1.24xlarge       {"commit": "37c8653", "provisioner": "default"}
+2022-10-28T08:24:45.136Z        INFO    controller.provisioning Waiting for unschedulable pods  {"commit": "37c8653"}
+```
+
+The inference pod should be scheduled on the node provisioned by Karpenter. Check if the Pod is in it's ready state:
+
+:::note
+It can take up to 8 minutes to provision the node, add it to the EKS cluster, and start the pod.
+:::
+
+```bash timeout=360
+$ kubectl -n aiml wait --for=condition=Ready --timeout=8m pod/inference
```

Comment:
After the pod is running can you use `kubectl get node` to show the `aws.amazon.com/neuron` resources on the node? I think that would help tie in the node to the pod request a bit better.